packages/components/src/AuthenticatedRoute.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { Route, RouteProps, Redirect } from "react-router-dom";
+import { getApolloSession } from "@joincivil/utils";
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+export interface AuthenticatedRouteProps extends RouteProps {
+  redirectTo: string;
+  onlyAllowUnauthenticated?: boolean;
+}
+
+const userQuery = gql`
+  query {
+    currentUser {
+      uid
+      email
+      ethAddress
+      onfidoApplicantId
+      onfidoCheckID
+      kycStatus
```

Comment:
We can probably remove `onfidoApplicantId` `onfidoCheckID` `kycStatus` since we no longer need kyc

Reply:
Removed.